python/lsst/ip/isr/isrTask.py

```diff
@@ -597,7 +599,16 @@ def getIsrExposure(self, dataRef, datasetType, immediate=True):
         try:
             exp = dataRef.get(datasetType, immediate=immediate)
         except Exception, e:
-            raise RuntimeError("Unable to retrieve %s for %s: %s" % (datasetType, dataRef.dataId, e))
+            try:
+                if not self.config.fallbackFilterName:
+                    raise ValueError("No fallback filter specified")
+                exp = dataRef.get(datasetType, filter=self.config.fallbackFilterName, immediate=immediate)
+                self.log.warn("Using fallback calibration from filter %s" % self.config.fallbackFilterName)
+            except ValueError as e:
+                raise RuntimeError("Unable to retrieve %s for %2: %s and not fallbackFilterName defined" %\
+                        (datasetType, dataRef.dataId, e))
+            except:
+                raise RuntimeError("Unable to retrieve %s for %s: %s" % (datasetType, dataRef.dataId, e))
```

Comment:
The error message here is identical to the error message for the `ValueError` clause so why does the `ValueError` clause exist?